packages/augur-sdk/src/utils.ts

```diff
@@ -134,32 +133,43 @@ export function calculatePayoutNumeratorsValue(
     }
```

Comment:
The line above this one also needs `outcome: null`

Reply:
updated